tests/whitebox/_misc.py

```diff
@@ -37,6 +38,32 @@
     sys.exit(message)
 
 
+def check_error(obj, expected_cause, command_line, expected_code):
```

Comment:
Here, you should document the ```obj``` parameter as well, since it is not ```self```. It's type is ```unittest.TestCase```.

Reply:
Can't we be more specific and have the type be SimTestCase? (I believe this question won't be relevant once we implement the new intermediary class)

Reply:
Since we're planning to make it an instance method of ```RunTestCase``` anyway, then its type will be ```RunTestCase```, so just ignore this comment.

Reply:
But just for my better understanding, if we were not implementing `RunTestCase`, but rather keeping `check_error` as an external function, it would be correct to have the `obj` type as `SimTestCase`?

Reply:
We would still use the more general type. There is nothing obviously inherent in this method that makes it only suitable for tests that would require the daemon to be running. You have one counter-example: look at ```testPropagate``` in PropagateTestCase.

Reply:
Understood, thank you!